include/rocksdb/statistics.h

```diff
@@ -448,6 +448,11 @@ enum Tickers : uint32_t {
   // # of bytes written into blob cache.
   BLOB_DB_CACHE_BYTES_WRITE,
 
+  // Number of bytes of compressed values in CompressedSecondaryCache.
+  COMP_SEC_CACHE_BYTES_COMPRESSED,
```

Comment:
I suggest calling them ```COMP_SEC_CACHE_BYTES_ADD_COMPRESSED``` and ```COMP_SEC_CACHE_BYTES_ADD_UNCOMPRESSED```. Otherwise, it may be confused with number of bytes inserted (compressed) and bytes looked up (uncompressed).

Reply:
I will use `COMP_SEC_CACHE_BYTES_READ_COMPRESSED` if we agree to obtain this stats in Lookup.